apps/admin/lib/partners/getScoutsForTopConnectorPartner.ts

```diff
@@ -0,0 +1,32 @@
+import { prisma } from '@charmverse/core/prisma-client';
+
+export async function getScoutsForTopConnectorPartner({ week }: { week: string }) {
+  const events = await prisma.builderEvent.findMany({
+    where: {
+      type: 'top_connector',
+      week
```

Comment:
mentioned above but i think this report should always give you the most recent data available at least. or maybe do this week and last week?

Reply:
Agree!